BridgeSDK/BridgeAPI/SBBUploadManager.m

```diff
@@ -113,8 +113,10 @@ - (instancetype)init
 
 - (NSURL *)tempUploadDirURL
 {
-    NSURL *uploadDir = nil;
+    NSURL *appSupportDir = [[[NSFileManager defaultManager] URLsForDirectory:NSApplicationSupportDirectory inDomains:NSUserDomainMask] firstObject];
+    NSString *bundleName = [[NSBundle mainBundle].infoDictionary objectForKey:@"CFBundleIdentifier"];
     NSString *tempUploadDirName = [NSTemporaryDirectory() stringByAppendingPathComponent:@"SBBUploadManager"];
+    NSURL *uploadDir = [[appSupportDir URLByAppendingPathComponent:bundleName] URLByAppendingPathComponent:tempUploadDirName];
     NSError *error;
     if ([[NSFileManager defaultManager] createDirectoryAtPath:tempUploadDirName withIntermediateDirectories:YES attributes:nil error:&error]) {
         uploadDir = [NSURL fileURLWithPath:tempUploadDirName];
```

Comment:
i think this is wrong, it will create the same directory as previously tmp/SBBUploadManager and that's the one that is going to be used, Library/Application Support is not used at all